backend/src/modules/db.js

```diff
@@ -1,5 +1,6 @@
 const {Pool} = require('pg')
 
+//.env declarations 
```

Comment:
Can we change this to `db connection` instead? The env declarations are handled in the docker compose file